x-pack/test/functional_with_es_ssl/apps/discover_ml_uptime/discover/search_source_alert.ts

```diff
@@ -399,6 +399,7 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
     });
 
     it('should navigate to alert results via link provided in notification', async () => {
+      await PageObjects.settings.refreshDataViewFieldList(OUTPUT_DATA_VIEW);
```

Comment:
I'm interested why this is failing when we don't refresh the output data view?

Reply:
I could dig deeper but the tests are creating data and data views, deleting them, and adding new content.